canairio/settings/testing.py

```diff
@@ -0,0 +1,19 @@
+from .base import *
```

Comment:
This is meant for unit testing ONLY. Any test outside of the realm of LIGHTNING FAST execution will happen in a different environment.